config.toml

```diff
@@ -13,6 +13,8 @@ region_name = "us-east-1"
 simple_model = "gpt-4o-mini"
 complex_model = "gpt-4o-2024-08-06"
 embeddings_model = "text-embedding-ada-002"
+base_url = "https://api.openai.com/v1/" # for openai compatible apis
+api_key = "" # for openai compatible apis, overrides the environment variable OPENAI_API_KEY
```

Comment:
I can't understand:
- if `api_key = ""` - will OPENAI_API_KEY be overwritten by ""? 
- the main purpose is to support local models with compatible api to openai, right? But then why do they require `api_key` if they are local?
- I'm afraid that storing `api_key` in a file that is under git will eventually result in leaking the key by someone doing `git commit -am "msg"`

Reply:
"" won't overwrite
The api key is used for eg ollama and in that case the key should be equal to "ollama". Local vendors specify what should be used for api_key in order to properly connect to endpoint.